hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/lock/LockManager.java

```diff
@@ -234,4 +234,17 @@ private void decrementActiveLockCount(final R resource) {
     });
   }
 
+  /**
+   * Returns the active lock count on a given resource.
+   *
+   * @param resource for which the active lock count has to be returned
+   * @return active lock count
+   */
+  public int getActiveLockCount(final R resource) {
+    ActiveLock activeLock = activeLocks.get(resource);
+    if (activeLock != null) {
+      return activeLock.getActiveLockCount();
+    }
+    return 0;
```

Comment:
Since the metrics is getting updated on `holdCount == 0`, please return -1 for null active lock case